lib/Bio/EnsEMBL/DataCheck/Checks/CheckIDRangeMLSS.pm

```diff
@@ -0,0 +1,132 @@
+=head1 LICENSE
+
+Copyright [2018-2021] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckIDRangeMLSS;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+use Bio::EnsEMBL::DataCheck::Utils qw( is_compara_ehive_db );
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'CheckIDRangeMLSS',
+  DESCRIPTION    => 'All relevant IDs are within the offset range for their MLSS ID',
+  GROUPS         => ['compara', 'compara_pairwise_alignments'],
+  DATACHECK_TYPE => 'advisory',
+  DB_TYPES       => ['compara'],
+  TABLES         => ['genomic_align', 'genomic_align_block', 'genomic_align_tree', 'constrained_element', 'conservation_score', 'dnafrag']
+};
+
+
+sub skip_tests {
+  my ($self) = @_;
+  if (is_compara_ehive_db($self->dba) != 1) {
```

Comment:
You want to skip the non-release databases, right?
```suggestion
  if (is_compara_ehive_db($self->dba) == 1) {
```

Reply:
I'd actually made this change and just forgotten to commit it before opening the PR